webroot/results/statistics.php

```diff
@@ -50,7 +50,7 @@ function showResults () {
   echo "<p style='padding-left:20px;padding-right:20px;font-weight:bold'>There are also three separate statistics pages: <a href='misc/evolution/'>Evolution of Records</a>, <a href='misc/sum_of_ranks/'>Sum of Ranks</a>, and <a href='misc/missing_averages/'>Missing Averages</a>.</p>";
   #--- TODO: Please remove the "New" label after 18 July 2017.
   echo "<p style='padding-left:20px;padding-right:20px;font-weight:bold'><span style='color: red;border: 2px solid red;padding: 2px;border-radius: 5px;font-size: 12px;'>New</span> You can find connections between cubers (aka Six Degrees) <a href='/relations'>here</a>.</p>";
-  echo "<p style='padding-left:20px;padding-right:20px;color:gray;font-size:10px'>Generated on " . wcaDate() . ".</p>";
+  echo "<p style='padding-left:20px;padding-right:20px;color:gray;font-weight:normal'>This page is automatically updated every 24 hours. Last Update: " . wcaDate() . ".</p>";
```

Comment:
My only comment is that it's unfortunate that this "24 hours" isn't connected to the [cronjob definition](https://github.com/thewca/worldcubeassociation.org/blob/9c0bee72df57a20dd96ac50fdd0d273ecc751ddb/chef/site-cookbooks/wca/recipes/cronjobs.rb#L51-L53) in any way. To use Lars's words: whenever someone changes the scheduling of that cron job, you just have to remember to update statistics.php. However, I don't think it's worth wiring that up here, as (someday) we're going to port this page to Ruby on Rails.

Reply:
Did you intentionally change the `font-size` here? Also, why the `font-weight: normal`? I don't believe this was bolded before.

Before image:

![image](https://user-images.githubusercontent.com/277474/34283363-2f379838-e680-11e7-8303-a7954001b365.png)

After image:

![image](https://user-images.githubusercontent.com/277474/34283372-388ec8fc-e680-11e7-974f-9ce8389a9c3c.png)

Reply:
Yes, the font size change (or rather making this the same font size than the other text) is intentional, to make this text more noticeable. Everything looks as I expected, yes.

I was not sure if "font-weight:normal" was needed, maybe this could also be omitted, but keeping it won't hurt I guess.